src/Microsoft.ApplicationInsights.AspNetCore/DiagnosticListeners/Implementation/HostingDiagnosticListener.cs

```diff
@@ -38,10 +39,24 @@ public void OnBeginRequest(HttpContext httpContext, long timestamp)
         {
             if (this.client.IsEnabled())
             {
-                httpContext.Features.Set(new RequestTelemetry());
+                RequestTelemetry requestTelemetry = new RequestTelemetry();
```

Comment:
There's no instances of RequestTelemetry in the aspnet repositories that end up calling the HostingDiagnosticListener. This seems to be the only place where RequestTelemetry is created and sent to ApplicationInsights.